### PR TITLE
Case 21893: Remove disabling of --url for Android

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/21893/Enable-the-url-command-line-parameter-on-Android

# Testing
1.  Verify that build runs on Android
2.  Verify that build runs on Quest
3.  Verify build runs on PC

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -3726,14 +3726,10 @@ void Application::handleSandboxStatus(QNetworkReply* reply) {
 
     // If this is a first run we short-circuit the address passed in
     if (_firstRun.get()) {
-#if !defined(Q_OS_ANDROID)
        DependencyManager::get<AddressManager>()->goToEntry();
        sentTo = SENT_TO_ENTRY;
-#endif
```
Comment (SamGondelman) [inline interface/src/Application.cpp]: Do you know why these were originally ifdefed out?  I wonder if this will break the 2D android app.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18346/HighFidelity-Beta-PR15254-da8d04e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18346/HighFidelity-Beta-Interface-PR15254-da8d04e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18346/HighFidelity-Beta-PR15254-da8d04e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18346/HighFidelity-Beta-Interface-PR15254-da8d04e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18349/HighFidelity-Beta-PR15254-47f4c25.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18349/HighFidelity-Beta-Interface-PR15254-47f4c25.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18349/HighFidelity-Beta-PR15254-47f4c25.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18349/HighFidelity-Quest-Beta-PR15254-47f4c25.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18349/HighFidelity-Beta-PR15254-47f4c25.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18349/HighFidelity-Beta-Interface-PR15254-47f4c25.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18356/HighFidelity-Beta-PR15254-ce10f6f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18356/HighFidelity-Beta-Interface-PR15254-ce10f6f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18356/HighFidelity-Beta-PR15254-ce10f6f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18356/HighFidelity-Beta-Interface-PR15254-ce10f6f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18361/HighFidelity-Beta-PR15254-cb199b7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18361/HighFidelity-Beta-Interface-PR15254-cb199b7.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18361/HighFidelity-Beta-PR15254-cb199b7.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18361/HighFidelity-Quest-Beta-PR15254-cb199b7.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18361/HighFidelity-Beta-PR15254-cb199b7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18361/HighFidelity-Beta-Interface-PR15254-cb199b7.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18362/HighFidelity-Beta-PR15254-0c74668.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18362/HighFidelity-Beta-Interface-PR15254-0c74668.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18362/HighFidelity-Beta-PR15254-0c74668.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18362/HighFidelity-Quest-Beta-PR15254-0c74668.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18362/HighFidelity-Beta-PR15254-0c74668.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18362/HighFidelity-Beta-Interface-PR15254-0c74668.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18363/HighFidelity-Beta-PR15254-c17b2e8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18363/HighFidelity-Beta-Interface-PR15254-c17b2e8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18363/HighFidelity-Beta-PR15254-c17b2e8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18363/HighFidelity-Quest-Beta-PR15254-c17b2e8.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18363/HighFidelity-Beta-PR15254-c17b2e8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18363/HighFidelity-Beta-Interface-PR15254-c17b2e8.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18457/HighFidelity-Beta-PR15254-17e177f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18457/HighFidelity-Beta-Interface-PR15254-17e177f.exe

**Pull request closed without merging**